lib/puppet/face/facts.rb

```diff
@@ -99,6 +98,7 @@
 
     $ puppet facts show os
     EOT
+    default true
```

Comment:
nit could you update the [`summary` on line 87](https://github.com/puppetlabs/puppet/pull/8402/files#diff-c5671b914e36764d426448e6380a7fe9eb7f088ed71590fb326c3384e31442caR87), it looks like it was copy/pasted from another file?

Reply:
I adjusted the summary and the description.